src/spawn/arguments.ts

```diff
@@ -0,0 +1,64 @@
+const _ = require('underscore');
+const checkTypes = require('check-types');
+
+import { CanDeployOptions } from '../can-deploy';
+import { MessageOptions } from '../message';
+import { PublisherOptions } from '../publisher';
+import { ServiceOptions } from '../service';
+import { VerifierOptions } from '../verifier';
+
+export type SpawnArguments =
+  | CanDeployOptions[]
+  | MessageOptions
+  | PublisherOptions
+  | ServiceOptions
+  | VerifierOptions
+  | {};
+
+export const DEFAULT_ARG = 'DEFAULT';
+
+export class Arguments {
+  public fromSpawnArgs(
```

Comment:
Is there a case that `fromSpawnArgs` should actually be `toSpawnArgs` because it's job is to create
the spawn arguments?

Reply:
Good catch! I changed it to `toArgumentsArray`, since it's taking `SpawnArguments` and turning them into `String[]`